rune/libenclave/intelsgx/aesmd.go

```diff
@@ -328,7 +328,7 @@ func GetQuote(report []byte, spid string, linkable bool) ([]byte, error) {
 	}
 
 	quote := resp.GetQuote.GetQuote()
-	if len(quote) < QuoteLength || len(quote) != SgxMaxQuoteLength {
+	if len(quote) < QuoteLength || len(quote) > SgxMaxQuoteLength {
```

Comment:
Why changing to `>`? Even the requested bufSize is longer than the actual size, the return value is still equal to the requested one.

Reply:
> Why changing to `>`? Even the requested bufSize is longer than the actual size, the return value is still equal to the requested one.

Because the bufSize in the `AesmServiceRequest_GetQuote` request is equal to `SgxMaxQuoteLength`, so the return length of Quote can't be greater than  `SgxMaxQuoteLength`.